app/controllers/admin_controller.rb

```diff
@@ -84,5 +95,23 @@ def export_header_str
 
   end
 
+  def paid_H_or_link (arg)
+    out_str = ''
+
+    if arg.company_id.nil?
+      out_str << '-'
+      out_str << ','
+    else
+      # say betald if branding fee is paid, otherwise makes link to where it is paid (when logged in)
+      out_str << (Company.find_by_id(arg.company_id).branding_license? ? 'Betald' : 'Betalas som inloggad via: http://hitta.sverigeshundforetagare.se' + company_path(Company.find_by_id(arg.company_id)))
```

Comment:
See comment above.  This (and related) statement(s) will _never_ find a company, since we are no longer using `company_id` in ShfApplication.  

Please note that a test of the export against seeded data shows this to be the case via the "Company" column.

<img width="1284" alt="screen shot 2018-01-26 at 9 21 33 am" src="https://user-images.githubusercontent.com/9968213/35443801-53c3f32a-027a-11e8-90ec-afed0fdb425a.png">

Reply:
Works fine here?

Reply:
Aha, I hadnt reset the db. I see. Back to the drawing board.